openlibrary/plugins/openlibrary/js/search.js

```diff
@@ -68,3 +96,41 @@ export function initSearchFacets() {
         less($(this).data('header'), start_facet_count, facet_inc);
     });
 }
+
+/**
+ * Makes call for a partially rendered facet counts sidebar, using the given search
+ * parameters.
+ *
+ * @param {Object} param
+ * @returns {Promise<Response>}
+ *
+ * @throws Will throw error if `/partials` response is not in 200-299 range.
+ */
+function fetchPartials(param) {
+    const data = {p: param}
```

Comment:
`p` is cryptic, let's just call it `param` to match the name of the argument to the actual template.